packages/pie-toolbox/src/code/plot/label.jsx

```diff
@@ -20,6 +20,7 @@ const LabelComponent = (props) => {
     side,
     onChange,
     mathMlOptions = {},
+    charactersLimit,
```

Comment:
here it can remain this name for the new prop

Reply:
Done